amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpRpcFlowStage.scala

```diff
@@ -53,14 +53,6 @@ final class AmqpRpcFlowStage(settings: AmqpSinkSettings, bufferSize: Int, respon
 
       override def whenConnected(): Unit = {
         import scala.collection.JavaConverters._
-        val shutdownCallback = getAsyncCallback[(String, Option[ShutdownSignalException])] {
```

Comment:
This was unnecessary since we already have the `shutdownCallback` in the `ConnectorLogic`. Updated the calls to invoke the right method.